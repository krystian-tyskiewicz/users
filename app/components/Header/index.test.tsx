import { render } from '@testing-library/react'
import Header from '.'

it('renders correctly', () => {
  const { container } = render(<Header>Test</Header>)
  expect(container).toMatchSnapshot()
})
