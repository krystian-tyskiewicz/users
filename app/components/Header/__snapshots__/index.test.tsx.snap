// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <header
    class="sc-bdvumH kXRUVr"
  >
    Test
  </header>
</div>
`;
